entity-service-impl/src/main/java/org/hypertrace/entity/query/service/converter/filter/NullFilteringExpressionConverter.java

```diff
@@ -10,18 +8,16 @@
 import org.hypertrace.core.documentstore.expression.operators.RelationalOperator;
 import org.hypertrace.core.documentstore.expression.type.FilterTypeExpression;
 import org.hypertrace.core.grpcutils.context.RequestContext;
-import org.hypertrace.entity.attribute.translator.EntityAttributeMapping;
 import org.hypertrace.entity.query.service.converter.ConversionException;
 import org.hypertrace.entity.query.service.converter.Converter;
-import org.hypertrace.entity.query.service.converter.identifier.IdentifierConverterFactory;
+import org.hypertrace.entity.query.service.converter.identifier.IdentifierExpressionConverter;
 import org.hypertrace.entity.query.service.v1.ColumnIdentifier;
 import org.hypertrace.entity.query.service.v1.LiteralConstant;
 import org.hypertrace.entity.query.service.v1.Operator;
 
 @AllArgsConstructor(onConstructor_ = {@Inject})
 public class NullFilteringExpressionConverter extends FilteringExpressionConverterBase {
-  private final EntityAttributeMapping entityAttributeMapping;
-  private final IdentifierConverterFactory identifierConverterFactory;
+  private final IdentifierExpressionConverter identifierExpressionConverter;
```

Comment:
Is this binding working? I see that IdentifierExpressionConverter is bound to `Converter<ColumnIdentifier, IdentifierExpression>` only @ IdentifierModule

Reply:
fair point. let me test this end to end

Reply:
tested this e2e. seems to be working fine